lib/girouette/src/girouette/tw/common.cljc

```diff
@@ -52,6 +52,25 @@
 (defn ratio-str [[x y]] (str x " / " y))
 
 
+;; This can be changed by passing another value in `make-api`
+(def ^:dynamic *default-number-unit* "rem")
```

Comment:
I would prefer to pass the `default-number-unit` property by parameter to the components.

Reply:
Not sure, but it seems that "rem" will get overwritten by the binding all the time.